src/MMM01.md

```diff
@@ -0,0 +1,403 @@
+```
+
+Note: It's suspected the lowest bit of ROM Bank Low (post $00->$01 remapping) still affects the bank mapped to the 4000-7FFF region in "unmapped" mode the same as it does in mapped mode.
+Most of the time this would still be a 1, but during game selection it could momentarily go to 0 in between setting the game select bits in [ROM Bank Low](<#Bits 0-4: ROM Bank Low>) and masking them as such in [ROM Bank Mask](<#Bits 1-5: ROM Bank Mask>).
+
+Mapped, multiplex disabled:
+
+```
+Bits: 22 21 20 19 18 17 16 15 14 13 12 .. 01 00
+      \___/ \___/ \____________/ \____________/
+        |     |          |            \----------- From Game Boy address
+        |     |          \------------------------ ROM Bank Low (complete value, not masked)
+        |     |                                     anti-masked $00 is remapped to $01 (see explanation above)
+        |     \----------------------------------- ROM Bank Mid
+        \----------------------------------------- ROM Bank High
+```
+
+### A000-BFFF - RAM Bank $00-03, if any (Read/Write)
+
+This area is used to address external SRAM in the cartridge (if any).
+The RAM is only accessible [if RAM is enabled](<#0000-1FFF - RAM Enable (Write Only)>), otherwise reads return open bus values (often $FF, but not guaranteed) and writes are ignored.
+
+External RAM is often battery-backed, allowing for the storage of game data while the Game Boy is turned off, or if the cartridge is removed from the Game Boy.
+
+It is currently unknown whether RAM access is possible while in unmapped mode.
+
+#### Addressing diagrams
+
+In mode 0:
+
+```
+Bits: 16 15 14 13 12 .. 01 00
+      \___/ \___/ \_________/
+        |     |        \-------- From Game Boy address
+        |     \----------------- bitwise and of RAM Bank Low & RAM Bank Mask
+        \----------------------- RAM Bank High
+```
+
+In mode 1:
+
+```
+Bits: 16 15 14 13 12 .. 01 00
+      \___/ \___/ \_________/
+        |     |        \-------- From Game Boy address
+        |     \----------------- RAM Bank Low (complete value, not masked)
+        \----------------------- RAM Bank High
+```
+
+## Registers
+
+The MMM01 registers are extensions of the MBC1 registers.
+In "mapped" mode, the extra bits cannot be written to, and the MMM01 emulates an MBC1.
+In "unmapped" mode, the extra bits are writeable, allowing the menu program to select which game to play and configure its size and RAM access.
+
+All the MMM01 registers are 7-bits in size and set to $00 on power-up.
+For the ROM Bank Number register, this behaves as if it was set to $01.
+
+### 0000-1FFF - RAM Enable (Write Only)
+
+```
+Bits: X 6 5 4 3 2 1 0
+        | \_/ \_____/
+        |  |      \----- Bits 0-3: RAM Enable
+        |  \------------ Bits 4-5: RAM Bank Mask
+        \--------------- Bit 6: Mapping Enable
+```
+
+#### Bits 0-3: RAM Enable
+
+As per MBC1, writing Ah to the lower 4 bits enables the external RAM, and any other value disables it.
```

Comment:
Hex values must be preceded (prefixed). Please see https://github.com/gbdev/pandocs/wiki/Document-Style